maven project/buy-a-car/src/main/java/com/retail/business/CarBusiness.java

```diff
@@ -0,0 +1,24 @@
+package com.retail.business;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.retail.shop.car.dto.CarDto;
+import com.retail.shop.car.model.Car;
+
+public class CarBusiness implements ICarBusiness {
```

Comment:
preferred class name could be CarBusinessImpl instead! It gives a quick idea that it is an implementation class for an abstract definition(Interface or abstract class)